sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -50,60 +188,52 @@ namespace Azure { namespace Core {
       /// RFC 1123.
       Rfc1123,
 
-      /// ISO 8601.
-      Iso8601,
+      /// RFC 3339.
+      Rfc3339,
     };
 
     /**
      * @brief Get the current UTC time.
      */
-    static DateTime UtcNow();
-
-    /// An invalid UTC timestamp value.
-    static constexpr IntervalType UtcTimestampInvalid = static_cast<IntervalType>(-1);
-
-    /**
-     * @brief Get seconds since Unix/POSIX time epoch at `01-01-1970 00:00:00`.
-     * If time is before epoch, @UtcTimestampInvalid is returned.
-     */
-    static IntervalType UtcTimestamp()
-    {
-      auto const seconds = UtcNow().ToInterval() / WindowsToPosixOffsetSeconds;
-      return (seconds >= WindowsToPosixOffsetSeconds) ? (seconds - WindowsToPosixOffsetSeconds)
-                                                      : UtcTimestampInvalid;
-    }
+    static DateTime Now();
 
     /**
-     * @brief Construct an uninitialized (!@IsInitialized()) instance of @DateTime.
+     * @brief Construct an instance of @DateTime.
+     *
+     * @param year Year.
+     * @param month Month.
+     * @param day Day.
+     * @param hour Hour.
+     * @param minute Minute.
+     * @param seconds Seconds.
+     *
+     * @throw DateTimeException If any paramter is invalid.
```

Comment:
Document the default date if I call: `DateTime(0)`

Reply:
That's fine. What's the default date if I only provide the year? For example `DateTime(2020)` - This creates 01-01-2020 00:00:00.00... correct? That might be helpful to document.

Reply:
Doxygen should be populating defaults. Too bad I can't check at the moment, as the build pipeline is broken.

Reply:
Doxygen does provide values for defaults: 
![image](https://user-images.githubusercontent.com/41349689/98893013-6d089b00-2456-11eb-835d-c84995b9fed6.png)

![image](https://user-images.githubusercontent.com/41349689/98893049-7f82d480-2456-11eb-951d-8d8f8ea4f70e.png)